Source/Core/Core/IOS/USB/Emulated/Microphone-Logitech.h

```diff
@@ -0,0 +1,34 @@
+// Copyright 2025 Dolphin Emulator Project
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+#pragma once
+
+#include "Common/CommonTypes.h"
+#include "Core/IOS/USB/Emulated/Microphone.h"
+
+namespace IOS::HLE::USB
+{
+class LogitechMicState;
+
+class MicrophoneLogitech final : public Microphone
+{
+public:
+  MicrophoneLogitech(const LogitechMicState& sampler, u8 index);
+  ~MicrophoneLogitech();
```

Comment:
This destructor doesn't do anything and can be removed. As-is it produces a warning.